### PR TITLE
Accept NumpadEnter as Enter

### PR DESCRIPTION
**Pull request by dimvoly**
Just a single line of code added to be able to use NumpadEnter. See  [Issue #170](https://github.com/mvlabat/bevy_egui/issues/170).

To get this to compile I did have to change the level of `missing_docs` as the compiler complains of the `WorldQuery` derive macro being undocumented. See bevy [Issue #3492](https://github.com/bevyengine/bevy/issues/3492), the crate `bevy_ecs_macros` is not yet documented enough. Once that is fixed, then the level of documentation can go back up to `deny` (under a separate PR).

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#![deny(missing_docs)]
+#![warn(missing_docs)]
```
Comment (mvlabat) [inline src/lib.rs:1]: Could you revert this change please? Other than that, looks good to me. Thanks!

**Review by mvlabat — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (mvlabat) [reply src/lib.rs:1]: On the second thought, I guess it'll make CI fail anyway. Let's keep it then :)

**Review by mvlabat — APPROVED** [review]

File: src/lib.rs (hunk 1 above)
Comment (mvlabat) [reply src/lib.rs:1]: Oh I see. It seems like the new Bevy version breaks CI as its macro generates a struct without documentation.

Comment (dimvoly) [reply src/lib.rs:1]: We could `#![allow(missing_docs)]`? See [SO thread](https://users.rust-lang.org/t/solved-applying-attributes-to-macro-included-code/30484/7). Not too sure how to exclude a single derive macro from the missing docs, there's some [forum posts here](https://users.rust-lang.org/t/solved-applying-attributes-to-macro-included-code/30484/7), but it looks messy and might only work for proc macros.

Could allow and wait for bevy to write their docs.
